sebs/cloudflare/cloudflare.py

```
@@ -0,0 +1,1608 @@
+            )
+
+        headers = self._get_auth_headers()
+
+        # Log credential type being used (without exposing the actual token)
+        if self.config.credentials.api_token:
+            token_preview = self.config.credentials.api_token[:8] + "..." if len(self.config.credentials.api_token) > 8 else "***"
+            self.logging.info(f"Using API Token authentication (starts with: {token_preview})")
+        else:
+            self.logging.info(f"Using Email + API Key authentication (email: {self.config.credentials.email})")
+
+        response = requests.get(f"{self._api_base_url}/user/tokens/verify", headers=headers)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)